playground/src/app.critical.scss

```diff
@@ -14,6 +14,21 @@
   -webkit-tap-highlight-color:  rgba(255, 255, 255, 0);
 }
 
+body * {
```

Comment:
Seems tempting but I'd vote to explicitly set this to the editor. The native scrollbars provide better UX for things like the error list.